sdk/examples/how_tos/native_tokens/create.rs

```diff
@@ -0,0 +1,84 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! In this example we will create a native token.
+//!
+//! Make sure that `example.stronghold` and `example.walletdb` already exist by
+//! running the `create_account` example!
+//!
+//! Rename `.env.example` to `.env` first, then run the command:
+//! ```sh
+//! cargo run --release --all-features --example create_native_token
+//! ```
+
+use std::env::var;
```

Comment:
:trollface:

Reply:
this is not the concern of this PR, but rather the alignment PR :wink:

Reply:
Yeah fine